src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: $calendar-width;
+  display: flex;
+
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  &__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $cell-color;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid #333;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    transition: 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
```

Comment:
1. The main font-family should be applied to the html or body element.
2. When you use transition specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/788ee22a-62db-4523-9254-1e7f9283f191)
3. Style cursor: pointer;  under general selector instead of :hover